src/InstrumentationEngine.ProfilerProxy/InstrumentationEngine.ProfilerProxy.vcxproj

```diff
@@ -65,7 +65,7 @@
   </ImportGroup>
   <ItemDefinitionGroup>
     <ClCompile>
-      <AdditionalIncludeDirectories>$(OutDir);..\Common.Lib;..\InstrumentationEngine.ProfilerProxy.Lib;..\..\inc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(OutDir);..\Common.Lib;..\InstrumentationEngine.ProfilerProxy.Lib;..\..\inc;..\InstrumentationEngine.Lib;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
>..\InstrumentationEngine.Lib [](http://example.com/codeflow?start=113&length=28)

Ugh... linking InstrumentationEngine.Lib into this dll isn't great. There is a lot of code in there that we don't need in this one. On Windows, I think that will be fine. The linker should strip out unused code for retail builds. I don't know what it will do on Linux, though.

Reply:
We could consider making StringUtils header-only (move all of the implementation into the .h file). That way, we wouldn't need to add this additional linkage.

Reply:
I can create a GitHub issue for that; currently the ProfilerProxy is only used in the MSI scenario but I can see it going xpat.

Reply:
[https://github.com/microsoft/CLRInstrumentationEngine/issues/417](https://github.com/microsoft/CLRInstrumentationEngine/issues/417)

Reply:
OK. I'll sign off on this for now, so that we can get it in and unblock our partners. I would like to see #417 fixed ASAP, though.